.env.example

```diff
@@ -0,0 +1 @@
+PINATA_JWT=
```

Comment:
Hmm...
We should not be having an env file of any sort inside a package 🤔

Reply:
It’s basically just for testing, not for the package .. you can check it has no use case inside any of the packages.. maybe the file shouldn’t be commited

Reply:
Yeah....I do understand. 
The next comment covers how you can test using values from the environment but without using an env file 🙇